arbitrum-docs/run-arbitrum-node/arbos-releases/01-overview.md

```diff
@@ -27,20 +27,28 @@ Visit [Inside Arbitrum Nitro](/how-arbitrum-works/inside-arbitrum-nitro.md) to l
+The number used to denote each upgrade will increment by 10, starting from ArbOS 20 (i.e. the next ArbOS upgrade after ArbOS 20 will be ArbOS 31). This was done because there are teams who have customized their Orbit chain's [behavior](/launch-orbit-chain/how-tos/customize-stf.mdx) or [precompiles](/launch-orbit-chain/how-tos/customize-precompile.mdx) and who may wish to use ArbOS's naming schema between official ArbOS version bumps (e.g. ArbOS 12 could be the name of a customized version of ArbOS for a project's L3 Orbit chain). 
+
+Note that there may be cases where special optimizations or changes are needed to be made for a specific family of ArbOS releases that will diverge from the standard numbering scheme described above. For example, ArbOS 31 will be the canonical ArbOS version for the “Bianca” family of releases and it is expected that node operators and chain owners upgrade from ArbOS 20 directly to ArbOS 31 (instead of ArbOS 30).
 
 ## Network status
 
 To view the status and timeline of network upgrades on Arbitrum One and Nova, [please visit this page](https://docs.arbitrum.foundation/network-upgrades).
 
+## Expectations for Orbit chain owners
+
+For Orbit chain owners or maintainers: it is important to note that _before_ upgrading your Orbit chain(s) to the newest ArbOS release, we strongly encourage waiting at least 2 weeks after the new ArbOS release has been activated on Arbitrum One and Nova before attempting the upgrade yourself. The rationale behind this short time buffer is to allow the Offchain Labs team to address any upgrade issues or stability concerns that may arise with the initial rollout so that we can minimize the chances of your chain(s) hitting the same or similar issues and to maximize the likelihood of an eventual smooth, seamless upgrade. Arbitrum Orbit chains, as always, can pick up new features & enable new customizations as they see fit. ArbOS upgrades, like ArbOs 31 "Bianca" that unlocks Stylus for use, is no exception to this rule. However, we believe this delay is worth it to ensure consistent UX across all Orbit chain owners and managers for these critical upgrades.
+
+Note that enabling an ArbOS upgrade is not as simple as bumping your chain’s Nitro node version. Instead, there are other steps required that are outlined in our docs on [How to upgrade ArbOS on your Orbit chain](../../launch-orbit-chain/how-tos/arbos-upgrade.md). Please be sure to follow them and let us know if you encounter any issues. 
```

Comment:
```suggestion
For Orbit chain owners or maintainers: it is important to note that _before_ upgrading your Orbit chain(s) to the newest ArbOS release, we strongly encourage waiting at least 2 weeks after the new ArbOS release has been activated on Arbitrum One and Nova before attempting the upgrade yourself. The rationale behind this short time buffer is to allow the Offchain Labs team to address any upgrade issues or stability concerns that may arise with the initial rollout so that we can minimize the chances of your chain(s) hitting the same or similar issues and to maximize the likelihood of an eventual smooth, seamless upgrade. Arbitrum Orbit chains, as always, can pick up new features & enable new customizations as they see fit. ArbOs 31 "Bianca" which unlocks Stylus for use, is no exception to this rule. However, we believe this delay ensures consistent UX across all Orbit chain owners and managers for these critical upgrades.

Note that enabling an ArbOS upgrade is not as simple as bumping your chain’s Nitro node version. Instead, there are other steps required that are outlined in our docs on [How to upgrade ArbOS on your Orbit chain](../../launch-orbit-chain/how-tos/arbos-upgrade.md). Please be sure to follow them and let us know if you encounter any issues. 
```